contracts/utils/Redeemer.sol

```diff
@@ -22,42 +22,48 @@ contract Redeemer {
     * It tries to redeem proposal rewards from the contribution rewards scheme.
     * This function does not emit events.
     * A client should listen to GenesisProtocol and ContributionReward redemption events
-    * to monitor redemption operations. 
+    * to monitor redemption operations.
     * @param _proposalId the ID of the voting in the voting machine
     * @param _avatar address of the controller
     * @param _beneficiary beneficiary
-    * @return result boolean array for the following redeem types:
-    *          result[0] -execute -true or false
-    *          result[1]- redeem reputation and stakingTokens(GEN) from GenesisProtocol
-    *          result[2] -redeem daoBounty(GEN) from GenesisProtocol
-    *          result[3]- reputation - from ContributionReward
-    *          result[4]- nativeTokenReward - from ContributionReward
-    *          result[5]- Ether - from ContributionReward
-    *          result[6]- ExternalToken - from ContributionReward
+    * @return gpRewards array
+    *          gpRewards[0] - stakerTokenAmount
+    *          gpRewards[1] - stakerReputationAmount
+    *          gpRewards[2] - voterTokenAmount
+    *          gpRewards[3] - voterReputationAmount
+    *          gpRewards[4] - proposerReputationAmount
+    * @return gpDaoBountyReward
+    * @return executed bool true or false
+    * @return crResult array
+    *          crResults[3]- reputation - from ContributionReward
```

Comment:
indices are wrong in comments for crResults

Reply:
done at 1a84f69